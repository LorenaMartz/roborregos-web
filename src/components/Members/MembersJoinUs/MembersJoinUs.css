.members-join-us {
	height: 100%;
	background: pink;
}