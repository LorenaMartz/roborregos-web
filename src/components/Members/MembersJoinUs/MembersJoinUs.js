import React, { Component } from 'react';
import './MembersJoinUs.css';

class MembersJoinUs extends Component {
  handleClick = () => {
    window.location.href = 'candidates';
    this.scrollToBottom();
  }

  render() {
    return (
      <div className='members-join-us'>
        <button 
          onClick={this.handleClick} 
          className='members-join-us-button members-join-us-button-contain members-join-us-button-text'>
              Join the Team
        </button>
      </div>
    );
  }
}

export default MembersJoinUs;
