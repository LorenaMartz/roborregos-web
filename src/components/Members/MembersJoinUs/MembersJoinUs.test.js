// @flow
import { getByTestId } from '@testing-library/react'
import React from 'react'
import { render, unmountComponentAtNode } from 'react-dom'
import { act } from 'react-dom/test-utils'
import renderer from 'react-test-renderer'

import MembersJoinUs from './MembersJoinUs'

let container = null
beforeEach(() => {
  // setup a DOM element as a render target
  container = document.createElement('div')
  if (document.body != null) {
    document.body.appendChild(container)
  } else {
    expect(document.body).not.toEqual(null)
  }
})

afterEach(() => {
  // cleanup on exiting
  unmountComponentAtNode(container)
  if (container != null) {
    container.remove()
  } else {
    expect(container).not.toEqual(null)
  }
  container = null
})

it('MemberÂ´s Join Us section renders correctly', () => {
  act(() => {
    render(<MembersJoinUs />, container)
  })
  expect(getByTestId(container, 'members-join-us')).not.toEqual(null)
  expect(getByTestId(container, 'members-join-us-button')).not.toEqual(null)
  expect(getByTestId(container, 'members-join-us-button')).toHaveTextContent('Join the Team')
})

it('Matches snapshot', () => {
  const tree = renderer.create(<MembersJoinUs />).toJSON()
  expect(tree).toMatchSnapshot()
})
