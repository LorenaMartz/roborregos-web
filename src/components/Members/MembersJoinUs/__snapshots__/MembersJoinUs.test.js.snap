// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches snapshot 1`] = `
<div
  className="members-join-us"
  data-testid="members-join-us"
>
  <button
    className="members-join-us-button members-join-us-button-contain members-join-us-button-text"
    data-testid="members-join-us-button"
    type="button"
  >
    Join the Team
  </button>
</div>
`;
